configure.sh

```diff
@@ -14,8 +13,9 @@ function usage()
 	echo "The following options can be set, with defaults specified in brackets:"
 	echo "  --target MODE         Sets the build mode. Supported build modes are"
 	echo "                        'debug' and 'release'. [${TARGET}]"
-	echo "  --circt-path PATH     Sets the path for the CIRCT tools and enables"
-	echo "                        building with CIRCT support. [${CIRCT_PATH}]"
+	echo "  --enable-hls PATH     Sets the path for the CIRCT tools and enables"
+	echo "                        building the HLS backend. [${CIRCT_PATH}]"
+	echo "                        The PATH should be absolut, i.e., not relative."
```

Comment:
Why does it need an absolute path? Is there no way to make a path absolute given a relative path?

Reply:
The starting point of a relative path becomes from where a command is executed.
So without an absolute path, I had to place the CIRCT build at different locations for the HLS tests to work.

Reply:
I decided to separate the enabling of HLS and the path to CIRCT into two flags.
`--enable-hls` is now used for enabling builds with the HLS backend.
`--circt-path` can be used to change the default location of where CIRCT is expected to be (i.e., the default location for scripts/build-circt.sh). An absolute path is still needed, but in most cases one wouldn't have to touch this and the full default path is shown when using `./configure.sh --help`.

Reply:
Okay, I just saw your comment now after I already did the review. I absolutely do not agree with splitting the two flags up. See other comment.